js/src/models/model.projects.js

```diff
@@ -279,17 +279,17 @@ Wu.Project = Wu.Class.extend({
 		var options = {
 			project : this.getUuid(),
 			access : projectAccess
-		}
+		};
 
 		// send request to API		
- 		Wu.Util.postcb('/api/project/addInvites', JSON.stringify(options), function (ctx, response) {
+ 		app.api.addInvites(options, function (ctx, response) {
```

Comment:
this returns `(err, result)` i think now, `ctx` is from old way.. @IgorZiegler  can you double-check and update?